src/compas_model/elements/element.py

```diff
@@ -98,6 +99,7 @@ def __init__(
         self._transformation = None
         self._worldtransformation = None
         self.features = []  # type: list[Feature]
+        self.material_name = None
```

Comment:
this is the name under which the material is stored in the model? which we said would have to be unique? user gets feedback if attempting to add a material with an existing name?